src/scenes/Languages/Edit/EditLanguage.tsx

```diff
@@ -0,0 +1,32 @@
+import { useSnackbar } from 'notistack';
+import * as React from 'react';
+import { FC } from 'react';
+import { Except } from 'type-fest';
+import { useUpdateLanguageMutation } from './EditLanguage.generated';
+import {
+  EditLanguageForm,
+  EditLanguageFormProps as Props,
+} from './EditLanguageForm';
+
+type EditLanguageProps = Except<Props, 'onSubmit'>;
+
+export const EditLanguage: FC<EditLanguageProps> = (props) => {
+  const [updateLanguage] = useUpdateLanguageMutation();
+  const { enqueueSnackbar } = useSnackbar();
+
+  const submit: Props['onSubmit'] = async (input) => {
+    const res = await updateLanguage({
+      variables: { input },
+      refetchQueries: ['LanguageList', 'Language'],
+    });
+    const language = res.data!.updateLanguage.language;
+
+    enqueueSnackbar(`Updated language: ${language.name.value}`, {
+      variant: 'success',
+    });
```

Comment:
Let's not use snackbars when changes can be directly seen by user (like when the dialog closes).

Reply:
Okay I wasn't sure because I was worried if the update didn't happen immediately.